notebooks/s3d-mil-nce-text-to-video-retrieval/s3d-mil-nce-text-to-video-retrieval.ipynb

```diff
@@ -54,11 +54,11 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    
</pre><p>Maybe hide it in the helper?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>